apsi_diploma/diploma_app/views.py

```diff
@@ -1,10 +1,57 @@
+from django.http import HttpResponse, HttpRequest
 from django.shortcuts import render
-
-from django.http import HttpRequest
+from .models import DissertationProcess
+import django_filters
 
 from allauth.account.decorators import login_required
 
 
 @login_required(login_url="/login")
 def home_page(request: HttpRequest):
     return render(request, "diploma_app/home.html")
+
+
+def repo(request):
+    processes = DissertationProcess.objects.filter(exam_grade__gte=3.0)
+
+    if "key_words" in request.GET:
+        keyords = request.GET["key_words"].split(",")
```

Comment:
typo (keyords instead of keywords). Not important, but just by the way we can fix this